CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
```

Comment:
```suggestion
nationality, personal appearance, race, religion, or sexual identity, and
```

Reply:
(As we are using the Oxford comma elsewhere, let's also use it here for consistency)

Reply:
good catch, I agree. Updated to use Oxford comma